### PR TITLE
traefik: Add plist for traefik service.

### PR DESCRIPTION
**Pull request by sklirg**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adds a plist entry (akin to the one for [elasticsearch](https://github.com/Homebrew/homebrew-core/blob/master/Formula/elasticsearch.rb#L86)) so traefik can be picked up as a service ran through Homebrew.

I'm unable to build from source due to problems with installing the `node-sass` during build. Installing node-sass works completely fine globally and in other projects where I use it, but in the brew build env it fails. Since I'm not changing the build itself, simply adding a plist file, it should not be related. Installing the bottle works just fine, and installs the plist as expected.

I also added a `revision` as I saw that in another commit which simply changes the formulae but doesn't update the bottle. If that's wrong, I'll revert it.

If I should change something, hit me up 👍

### CODE REVIEW

**Comment by sklirg** [comment]
Hi @MikeMcQuaid , sorry to tag you for this. 

This PR has been open for 14 days and I'm unsure if I'm missing something or I've done something wrong, or if the PR queue simply is so full that it takes time to go through all. :-) Would appreciate some feedback!

Best regards.

**Comment by SMillerDev** [comment]
As far as I know this wouldn't need a revision bump to be picked up. Other than that it looks good.

File: Formula/traefik.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Traefik < Formula
   url "https://github.com/containous/traefik/releases/download/v1.7.6/traefik-v1.7.6.src.tar.gz"
   version "1.7.6"
   sha256 "2fc433c81bace17e801a071fed1ae4405835a8adbc9d6969baa42bab61dbad9e"
+  revision 1
```
Comment (MikeMcQuaid) [inline Formula/traefik.rb]: Can remove this

**Review by MikeMcQuaid — COMMENTED** [review]
One tweak otherwise looking good 👍

File: Formula/traefik.rb (hunk 1 above)
Comment (sklirg) [reply Formula/traefik.rb]: Fixed 👍

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @sklirg!

**Pull request merged**